### ISSUE TITLE
there is a problem processing the backspace in mobile.

### ISSUE THREAD

**Issue by largeden** [froala/wysiwyg-editor#806]
korean languages and japanese languages there is a problem processing the backspace in iPhone and iPad.
(ver IOS 9.0.2)

RC3 is the same look.

https://www.dropbox.com/s/coizzp57bvjo1la/backspace.mov?dl=0

In another editor, no problem(CKeditor)
https://www.dropbox.com/s/rsstzeabvey5ytc/backspace_ckeditor.mov?dl=0

**Comment by stefanneculai** [comment]
Could you please write here the keys you're hitting. It's pretty hard to follow them on the video.

**Comment by largeden** [comment]
Sorry.

# Step1. Add keyboard in IOS set below.
![screen shot 2015-10-02 at 21 01 06](https://cloud.githubusercontent.com/assets/5575931/10246106/3537c5ca-6949-11e5-98e8-79cc22f9300c.png)

### Japanese - Romaji Mode
### korean Standard Mode


# Step2. Japanese language
![screen shot 2015-10-02 at 21 01 41](https://cloud.githubusercontent.com/assets/5575931/10246147/91a4a6a2-6949-11e5-9c16-174204b81a9c.png)

### Please input the following at froala editor.

### 1) aa -> backspace -> a
### 2) aaaaaaa -> backspace x 4 -> enter key or a



# Step3. Korean langauge
![screen shot 2015-10-02 at 21 01 28](https://cloud.githubusercontent.com/assets/5575931/10246180/cda24f2e-6949-11e5-891a-cabcca606611.png)

### Please input the following at froala editor.

### 1) ㄱ + ㅏ + ㄴ + ㅏ -> backspace -> ㄴ + ㅏ
### 2) ㄱ + ㅏ + ㄴ + ㅏ -> backspace x 2 -> 나 + ㅏ

Korean languages, there is another problem.

https://www.dropbox.com/s/58w1pafxlzq3uri/Korean.mov?dl=0

Please write in the way you used to have Korean test.
And try the Backspace key test.
You'll be able to see that mobile and laptops are different.

I believe the perfect editor if you gonna solve this problem. :)
I'll find a way to resolve the issue.

**Comment by stefanneculai** [comment]
@largeden, could you please test this issue again with the latest commit? If it fixes the issue please feel free to close it. Thank you!

**Comment by largeden** [comment]
@stefanneculai 
Thank you!! :p 

Backspace problem was resolved.

But... 
This problem occurs when press the Enter key.
Similar problems with backspace key.

This problem only occurs on Korean input.

https://www.dropbox.com/s/5xc649tssftq4r5/typing_enter.mov?dl=0

ps:
I'll take the test again in other OS and browser. Please a little more time.

**Comment by stefanneculai** [comment]
Last commit should fix it.

**Comment by largeden** [comment]
@stefanneculai 
great! perfect! :+1:

**Comment by stefanneculai** [comment]
@largeden  If there is anything else just let us know. Thanks for testing this. :+1:
